content/docs/comparison/high-level-controllers.md

```diff
@@ -3,9 +3,19 @@ title: vs. High Level Frameworks
 type: docs
 ---
 
-# vs High Level Frameworks
-
-KUDO vs. Other High Level Controller Frameworks (metacontroller)
+# KUDO vs High Level Frameworks
+High Level Frameworks are the area of tools that sit between simple configuration and fully fledged custom controllers.
+These Tools usually hide the complexities of the Kubernetes internals and allow more control than 
+simple Kubernetes operations. As they do not expose all details, the don't allow as much possibilities as full 
+custom controllers.
 
 ## Metacontroller
-TODO
+Metacontroller is a framework that enables the development of controllers based on web hooks. It makes it easy to
+define behavior for custom resources, or to add functionality to existing APIs.
+
+By using web hooks in any language, metacontroller covers a wider range of use cases than KUDO. 
```

Comment:
Wouldn't make a negative comparison here around use cases, more along the lines of - metacontroller can work with existing tools such as jsonnet, but requires developers to run the services for their webhooks as well as metacontroller itself.